### DISCUSSION TITLE
How to change placeholder of select type filter?

### DISCUSSION THREAD

**Discussion by dnjsgur0629** [KevinVandy/material-react-table#256] [Q&A] [answered]
```js
return (
    <MaterialReactTable
      columns={mockColumns}
      data={mockData}
      muiTableHeadCellFilterTextFieldProps={{
        placeholder: "yeah"
      }}
    />
  );
  ```

![image](https://user-images.githubusercontent.com/65391273/207755953-ffda58d1-b3ab-4f7a-97d2-f276178bc720.png)

It seems that can't change placeholdwr with `muiTableHeadCellFilterTextFieldProps`.
how can i change it?

[![Edit select-filter-placeholder](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/select-filter-placeholder-mukdhu?fontsize=14&hidenavigation=1&theme=dark)

**Comment by KevinVandy** [comment, marked as answer]
Mui Select Textfields are weird with placeholders. This might be an area for improvement where the library can make the code you suggested just work.

**Reply by KevinVandy** [reply]
try out v1.4.3

**Reply by dnjsgur0629** [reply]
thank you so much
